<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HouseKeeperClient</class>
 <widget class="QWidget" name="HouseKeeperClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HouseKeeperClient</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#HouseKeeperClient
{
border-image: url(:/new/prefix1/pictures/BGP.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,27,1">
   <item>
    <widget class="QFrame" name="headline_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">#headline_frame
{
background-color: rgba(0, 0, 0, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,1,2,4,1,4,0,1,4,4">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255,255);
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>USB连接状态 </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="usb_connect_state">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/new/prefix1/pictures/断开链接.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255,255);
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>当前网络状态 </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="net_status">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#net_status{border-image: url(:/new/prefix1/pictures/网络断开.png);}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255,255);
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>返回首页</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="return_home">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:enabled
{
	border-image: url(:/new/prefix1/pictures/返回.png);
}

QPushButton:hover
{
	border-image: url(:/new/prefix1/pictures/返回_green.png);
}

</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>65</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255,255);
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>当前时间 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="now_time">
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255,255);
font: 75 16pt &quot;Microsoft YaHei UI&quot;;</string>
          </property>
          <property name="text">
           <string>2018-04-06 13:24</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,40,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0,0,0,0);</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,8,1" columnstretch="1,8,1">
         <item row="0" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>66</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>126</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,1,40">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="20,1,40">
             <item>
              <widget class="QPushButton" name="hardware_download">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">
QPushButton:enabled
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(127, 101, 255, 255);
}

QPushButton:hover
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(127, 101, 255, 220);
}
</string>
               </property>
               <property name="text">
                <string>固件烧写</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="dev_manage">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">
QPushButton:enabled
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(47, 148, 255, 255);
}

QPushButton:hover
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(47, 148, 255, 220);
}</string>
               </property>
               <property name="text">
                <string>设备管理</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>18</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="30,1,20">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout" stretch="20,1,20">
               <item>
                <widget class="QPushButton" name="devid_entry">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton:enabled
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(255, 112, 30, 255);
}
QPushButton:hover
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(255, 112, 30, 220);
}</string>
                 </property>
                 <property name="text">
                  <string>终端ID录入</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>18</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="factory_test">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton:enabled
{
    border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(148, 206, 39, 255);
}
QPushButton:hover
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(148, 206, 39, 220);
}</string>
                 </property>
                 <property name="text">
                  <string>出厂测试</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="upgrade_package">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:enabled
{
	border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(189, 0, 189,255);
}
QPushButton:hover
{
    border-radius: 10px;
	color: rgb(255, 255, 255);
	font: 75 22pt &quot;Microsoft YaHei UI&quot;;
	background-color: rgba(189, 0, 189,220);
}</string>
               </property>
               <property name="text">
                <string>升级包制作</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>126</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>66</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>headline_frame</zorder>
  <zorder>verticalSpacer_6</zorder>
  <zorder>verticalSpacer_5</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
